src/containers/UserProfile/index.jsx

```diff
@@ -2,32 +2,43 @@ import { connect } from 'react-redux';
 
 import UserProfile from '../../components/UserProfile';
 import {
-  saveUserProfile,
-  saveUserProfilePhoto,
-  deleteUserProfilePhoto,
+  fetchProfile,
+  saveProfile,
+  saveProfilePhoto,
+  deleteProfilePhoto,
   openEditableField,
   closeEditableField,
 } from '../../actions/profile';
 
-const mapStateToProps = state => ({
-  currentlyEditingField: state.profile.currentlyEditingField,
-  saveState: state.profile.saveState,
-  savePhotoState: state.profile.savePhotoState,
-  error: state.profile.error,
-  profileImage: state.userAccount.profileImage.imageUrlLarge,
-  fullName: state.userAccount.name,
-  username: state.userAccount.username,
-  userLocation: state.userAccount.country,
-  education: state.userAccount.levelOfEducation,
-  socialLinks: state.userAccount.socialLinks,
-  bio: state.userAccount.bio,
-  certificates: null,
-});
+const mapStateToProps = (state) => {
+  const profileImage =
+    state.profilePage.profile.profileImage != null
```

Comment:
This was blowing up for me without this conditional check, since profileImage didn't exist in the reducer by default.  I think this is a better way of handling it than having a sparsely populated skeleton in the reducer - I recall being bitten by trying it that way in the past...